CMakeLists.txt

```
@@ -94,6 +94,8 @@ option ( enable-dsound "compile DirectSound support (if it is available)" on )
 option ( enable-wasapi "compile Windows WASAPI support (if it is available)" on )
 option ( enable-waveout "compile Windows WaveOut support (if it is available)" on )
 option ( enable-winmidi "compile Windows MIDI support (if it is available)" on )
+option ( enable-haiku "compile Haiku MediaKit audio support (if it is available)" on )
```

```suggestion
option ( enable-mediakit "compile Haiku MediaKit audio support (if it is available)" on )
```

IMO, the naming is unfortunate. `enable-haiku` would enable the audio driver and not support for Haiku in general. I'd suggest to rename it `mediakit`.